app/src/main/java/com/mapbox/maps/testapp/examples/WidgetActivity.kt

```diff
@@ -0,0 +1,85 @@
+@file:OptIn(MapboxExperimental::class)
+
+package com.mapbox.maps.testapp.examples
+
+import android.animation.ValueAnimator
+import android.content.Context
+import android.graphics.BitmapFactory
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.mapbox.geojson.Point
+import com.mapbox.maps.*
+import com.mapbox.maps.plugin.attribution.attribution
+import com.mapbox.maps.plugin.compass.compass
+import com.mapbox.maps.plugin.logo.logo
+import com.mapbox.maps.plugin.scalebar.scalebar
+import com.mapbox.maps.renderer.widget.BitmapWidget
+import com.mapbox.maps.renderer.widget.Widget
+import com.mapbox.maps.renderer.widget.WidgetPosition
+
+/**
+ * Adding a widget to the Map and demonstrating setRotation and setTranslation.
+ */
+class WidgetActivity : AppCompatActivity() {
```

Comment:
while the widget is not primarily targeted for MapView, wonder if it's necessary to add an example activity to demonstrate the use case.

Shall we just move this activity to instrumentation test?

Reply:
and then we just copy it back if we want to use it, or how would we launch this activity from tests?

Reply:
@paulrenn67 can we actually add the widget in `SurfaceActivity` instead of creating the new one? I agree with @pengdev that using OpenGL widgets with the `MapView` is an extremely rare use-case and it does not deserve it's own example activity. However using widgets with the `MapSurface` is the valid use-case and that's why we have `SurfaceActivity`

Reply:
ok good plan

Reply:
Have moved the `Widget` to `SurfaceActivity` and deleted `WidgetActivity`.